정택현/d01_0831.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+// MARK:- 완주하지 못한 선수
+
+var participants = ["mislav", "stanko", "mislav", "ana"]
+var completions = ["stanko", "ana", "mislav"]
+
+func solution(participant: [String], completion: [String]) -> String {
+    var ans = ""
+    var dic = [String: Int]()
+    
+    for p in participant {
+        dic[p] = (dic[p] ?? 0 ) + 1
+    }
+    
+    for c in completion {
+        dic[c] = (dic[c] ?? 0) - 1
+    }
+    
+    dic.forEach{ key,value in
+        if value > 0 { ans += key; ans += " "}
```

Comment:
`ans += " "`는 왜 해주신 걸까요?👀 문제 요구사항에 **completion의 길이는 participant 길이보다 1만큼 작다**고 했으니, 1명 이상은 나오지 않을 거에요!
```suggestion
        if value > 0 { ans += key }
```